src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -183,7 +162,7 @@ const NavHeader = (props) => {
                       ? {uri: topicDetail?.icon_path}
                       : TopicDefaultIcon
                   }
-                  style={styles.image}
+                  style={styles.image(followType)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dynamic style assignment based on `followType` is a good way to apply different styles conditionally. Just ensure that the `styles.image` function handles all possible values of `followType`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->